packages/services/emails/src/environment.ts

```diff
@@ -56,6 +56,7 @@ const SMTPEmailModel = zod.object({
   EMAIL_PROVIDER_SMTP_PORT: NumberFromString,
   EMAIL_PROVIDER_SMTP_AUTH_USERNAME: zod.string(),
   EMAIL_PROVIDER_SMTP_AUTH_PASSWORD: zod.string(),
+  EMAIL_PROVIDER_SMTP_REJECT_UNAUTHORIZED: zod.booleanValue
```

Comment:
```suggestion
  EMAIL_PROVIDER_SMTP_REJECT_UNAUTHORIZED: zod.boolean().optional()
```

According to https://zod.dev/?id=booleans it should be `boolean()`. I also added `optional()` as it's required only in specific cases.